src/Components/Book.js

```diff
@@ -11,28 +11,31 @@ function Book(props) {
   };
 
   return (
-    <div className="book-placeholder">
+    <div className="book-list">
       <div>
-        <span className="category-placeholder">Category</span>
+        <span className="category">Category</span>
         <h2>{title}</h2>
-        <span className="author-placeholder">{author}</span>
-        <div className="buttons">
+        <span className="author">{author}</span>
+        <div className="btn">
           <button type="submit">Comments</button>
           <button onClick={handleSubmit} type="submit">Remove</button>
           <button type="submit">Edit</button>
         </div>
       </div>
-      <div className="book-details">
-        <div className="chart-col">
-          <div className="pie" />
-          <div className="chart-desc">
-            <span className="perc">28%</span>
-            <span className="status">Not completed</span>
+      <div className="reading-preview">
+        <div className="pre">
+          <div className="progress" />
+          <div className="reading-desc">
+            <div className="oval" />
+            <div className="p-container">
+              <span className="percent-complete">78%</span>
```

Comment:
- [ ] the Circle is a little bit bigger than the one in the Design so it would be better if you make it a little bit smaller as the one in the Design in order to match the Desgin 
- Yours 
![Screenshot 2022-11-18 at 16 39 31](https://user-images.githubusercontent.com/61790587/202743989-eed9588e-b473-4617-934b-074d6ba4267b.png)
- Design 
<img width="116" alt="Screenshot 2022-11-18 at 16 40 32" src="https://user-images.githubusercontent.com/61790587/202744019-1d377f7a-dff7-4adc-9b85-0ff288670dc3.png">